docs/example/multi-interfaces-annotation/multus-conf.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: macvlan-conf-ens192
+  namespace: kube-system
+spec:
+  config: |-
+    {
+        "cniVersion": "0.3.1",
+        "name": "macvlan-conf-ens192",
+        "plugins": [
+            {
+                "type": "macvlan",
+                "master": "ens192",
+                "mode": "bridge",
+                "ipam": {
+                    "type": "spiderpool"
+                }
+            },{
+                  "type": "veth",
+                  "service_cidr": ["10.96.0.0/12"]
+              }
+        ]
+    }
+---
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: macvlan-conf-ens224
+  namespace: kube-system
+spec:
+  config: |-
+    {
+        "cniVersion": "0.3.1",
+        "name": "macvlan-conf-ens224",
+        "plugins": [
+            {
+                "type": "macvlan",
+                "master": "ens224",
+                "mode": "bridge",
+                "ipam": {
+                    "type": "spiderpool"
+                }
+            },{
+                  "type": "veth",
+                  "service_cidr": ["10.96.0.0/12"]
+              }
+        ]
+    }
```

Comment:
This code patch adds two Kubernetes NetworkAttachmentDefinition objects, which define configurations for network interfaces in the Kubernetes cluster. The first object defines a macvlan interface named "macvlan-conf-ens192" that uses the "ens192" interface as its master and sets its IP address allocation to use "spiderpool." It also includes a veth interface with service CIDR "10.96.0.0/12". The second object is similar, but creates a macvlan interface named "macvlan-conf-ens224" that uses "ens224" as its master.

Overall, the code looks fine, but some possible improvements are:
- Adding comments to explain the configuration of each interface may make it easier to understand and maintain the code in the future.
- Consider using variables or constants instead of hard-coded strings to improve readability and reusability of the code.
- Adding validation or error-checking mechanisms to ensure the code runs without issues would be helpful.